### PR TITLE
New Timer App with Brand Fetching Features.

### PR DESCRIPTION
**Pull request by salmanfarisvp**
Update the new timer app with new brand features. 

image

- [x] Update the colours as per the brand's primary and secondary colours 
- [x] Update the Brand logo and apply it to the timer app
- [x] Override time zone  
- [x] Override locale setting for region languages in dates. 
- [x] Support Standard Screen Sizes - As per [Supported Screen Resolutions](https://github.com/Screenly/Playground/blob/master/docs/resolutions.md)
- [x] Tracks/captures errors using [Sentry](https://www.sentry.io/)
- [x] Added Signal Ready  
- [x] QA done by at least two or more members.

### CODE REVIEW

File: edge-apps/countdown-timer/index.html [hunk 1]
```diff
@@ -1,88 +1,62 @@
...
+				</div>
+					<div class="secondary-card">
+					<object id="svgObject1" class="clock-icon" data="./static/img/clock-icon.svg" type="image/svg+xml"></object>
+					<span class="secondary-card-number second">0</span>
+					<span class="secondary-card-label">SECONDS</span>
+				</div>
+			</div>
+			<div class="row-container">
+				<div class="secondary-card info-card">
+					<img id="brandLogo" src="" class="brand-logo" alt="Brand Logo" />
```
Comment (iamplugged) [inline edge-apps/countdown-timer/index.html:40]: Can follow the hypen case for consistency id="brand-logo"

File: edge-apps/countdown-timer/index.html [hunk 2]
```diff
@@ -1,88 +1,62 @@
...
+				</div>
+			</div>
+		</div>
+	</div>
+
+	<script src="static/js/country-locale-map.1.18.5.min.js"></script>
+	<script src="static/js/moment-with-locales.min.js"></script>
+	<script src="static/js/moment-timezone-with-data.min.js"></script>
+	<script src="static/js/offline-geocode-city-1.0.2.min.js"></script>
+	<script src="static/js/tz.min.js"></script>
```
Comment (iamplugged) [inline edge-apps/countdown-timer/index.html:57]: Is tz.min.js library different from moment-timezone? Do we need to use both?

**Comment by iamplugged** [comment]
@salmanfarisvp A couple of comments. Looks good otherwise.

File: edge-apps/countdown-timer/index.html (hunk 1 above)
Comment (salmanfarisvp) [reply edge-apps/countdown-timer/index.html:40]: Sure, We can do that.

File: edge-apps/countdown-timer/index.html (hunk 2 above)
Comment (salmanfarisvp) [reply edge-apps/countdown-timer/index.html:57]: Hi @iamplugged, The **tz.min.js** fetches the timezone as per the latitude latitude when override_timezone values are invalid. 

And in **moment-timezone**, it manages the locale time and date information for "Language-specific formatting".

**Review by iamplugged — APPROVED** [review]

File: edge-apps/countdown-timer/index.html (hunk 2 above)
Comment (salmanfarisvp) [reply edge-apps/countdown-timer/index.html:57]: @iamplugged is it ok? Thanks.

Comment (iamplugged) [reply edge-apps/countdown-timer/index.html:57]: @salmanfarisvp Yes I've approved it.

**Comment by salmanfarisvp** [comment]
I found an issue with CSS on RPI screens and am working on the fix now.

**Comment by salmanfarisvp** [comment]
> I found an issue with CSS on RPI screens and am working on the fix now.

It was a chrome issue, not the code

**Comment by salmanfarisvp** [comment]
@iamplugged Changed brand logo fetching logic for svg vs jpeg/png and proxy url and plain url. 

I tested with different accounts and able to fetch the colour and logo  and fall back logo also working as expected.

**Review by iamplugged — APPROVED** [review]

**Pull request merged**